packages/runner/lib/state.ts

```diff
@@ -1 +0,0 @@
-export const abortControllers = new Map<string, AbortController>();
```

Comment:
instead of keeping 2 piece of states. Here and in `Monitor` I decided `Monitor` will be the place where worker references will be kept